modules/microshift-firewall-apply-settings.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-firewall-applying-settings_{context}"]
+== Applying firewall settings
+After you have finished configuring, run the following command to restart the firewall and apply settings:
+
+[source,terminal]
+----
+$ sudo firewall-offline-cmd --reload
+----
+
+//Q: How do we verify? What should we see after running this command?
```

Comment:
To verify, run the following cmds:
```
$ sudo firewall-cmd --list-all
$ sudo firewall-cmd --zone=trusted --list-all
```
The first cmd verify rules added in the default public zone (such as the ports related rules), the second cmd verify rules added in the trusted zone (such as the ip range related rules).